### ISSUE TITLE
Fix `hue-degree-notation` false negatives for `oklch`

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#7014]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
	color: lch(50% 50% 50);
	color: oklch(50% 50% 50);
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "hue-degree-notation": "angle"
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLpANhAE7wr5gAWAFAKwAMApCvU-QJQDcWuBxpEAa3LUWzRmM5YAviAA0IAGYBLfDABySALZxEMAB5aADqoB0YAM7m54CJmUBzBCAyYUKdCCIBXVeY+kXNzcPCi8YAFpYeyIYCMwIHCQcJVt-d2RMe1UPLDcpaWtIOyV7ADFiTSSnACtzW2tYQytEQPTzHABPVXwlTBw0j3wkmHaPWVy2zu7enHCih3D2pEwoJCIoAZAhnBH+kAKpIA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

I expected an error on `oklch`

### What actually happened?

No error was reported

### Do you have a proposal to fix the bug?

Add `oklch` in the same way as `lch` already exists today.
